forte/pipeline.py

```diff
@@ -466,6 +470,28 @@ def save(self, path: str):
         with open(path, "w", encoding="utf-8") as f:
             yaml.safe_dump(self._dump_to_config(), f)
 
+    def export(self, name: str = "pipeline") -> Optional[str]:
+        r"""Export pipeline to FORTE_EXPORT_PATH
+        FORTE_EXPORT_PATH is an environment variable
+        The pipeline is saved by :meth:`~forte.pipeline.Pipeline.save`
```

Comment:
Should make the docstring reflect the following behaviors:
1. The directory will be created if not found
2. A default name will be used to store the output
3. Raise exception when provided name is taken